src/coop/rchain/lang/folding/RholangFoldingBuilder.java

```diff
@@ -0,0 +1,79 @@
+package coop.rchain.lang.folding;
```

Comment:
Please convert the file to Kotlin, usualy it is much shorter than Java file. (Idea has a Convert command inside Code menu)

Reply:
Amazing!